src/test/system.rs

```diff
@@ -179,6 +183,16 @@ fn run_sccache_command_test(sccache: &Path, compiler: Compiler, tempdir: &Path)
         let expected_lines: Vec<_> = expected.lines().collect();
         assert_eq!(lines, expected_lines);
     }
+    if name == "gcc" {
```

Comment:
clang doesn't actually fail in this situation though, so it would be harder to write a useful test for it.